src/RepoAuditor/EntryPoint.py

```diff
@@ -181,6 +181,7 @@ def _VersionCallback(value: bool) -> None:
 )
 def EntryPoint(
     ctx: typer.Context,
+    *,

```

Comment:
cc @davidbrownell: I made most arguments into keyword arguments.
I tried to find out where and how this code gets executed, but it was not clear at all to me.

I was able to build and execute the CL executable with `uv build` then `uv run repo_auditor` locally, so I am assuming this change didn't break everything 🤞